essentials-framework/Essentials DM/Essentials_DM/Endpoints/Transmitters/DmTx200Controller.cs

```diff
@@ -33,6 +33,8 @@ public class DmTx200Controller : DmTxControllerBase, ITxRouting, IHasFeedback, I
         public IntFeedback VideoSourceNumericFeedback { get; protected set; }
         public IntFeedback AudioSourceNumericFeedback { get; protected set; }
         public IntFeedback HdmiInHdcpCapabilityFeedback { get; protected set; }
+        public BoolFeedback In1VideoSyncFeedback { get; protected set; }
+        public BoolFeedback In2VideoSyncFeedback { get; protected set; }
```

Comment:
Please change these names to reflect the type of input it is. HdmiIn1VideoSyncFeedback, etc.